docs/datastore/model-api.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Seems like we can merge this information with stuff above have single chapter.
WDYT?

Reply:
I think this stuff belongs to API reference section. That's how I see it.

Reply:
I would not have api reference yet. Model docs are really basic now so I was thinking to add more stuff there. Reference makes no sense if you have alpha version of the project.